scripts/zones/Valkurm_Dunes/mobs/Beach_Monk.lua

```diff
@@ -0,0 +1,73 @@
+-----------------------------------
+-- Area: Valkurm Dunes
+--  Mob: Beach Monk
+-- Spawned During Pirates Chart Quest
+-----------------------------------
+local ID = require("scripts/zones/Valkurm_Dunes/IDs")
+require("scripts/globals/status")
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.EXP_BONUS, -100)
+    mob:setMobMod(xi.mobMod.NO_DROPS, 1)
+    mob:setMobMod(xi.mobMod.CHARMABLE, 0)
```

Comment:
I don't think you'll need a few of these mods. Especially if you give them 34 mobType (notorious + event) 
I could be mistaken, but I think event monsters don't give any EXP. And I see he already has notorious flag, so he should already be uncharmable and drop no gil.

Reply:
NMs can definitely drop gil, maybe its the event mobtype? I have yet to use that one, I'll give it a try later!

Reply:
Yeah I wasn't 100% sure myself. Just a hunch lol

Reply:
Event mobtype did not prevent the mob from giving exp or gil. I removed charmable mod though.